package.json

```diff
@@ -66,8 +66,10 @@
     "@aws-sdk/client-sfn": "^3.358.0",
     "@aws-sdk/credential-providers": "^3.358.0",
     "@aws-sdk/property-provider": "^3.357.0",
+    "@google-cloud/logging": "^10.5.0",
     "@google-cloud/run": "^0.6.0",
     "@types/datadog-metrics": "0.6.1",
+    "agent-base": "^7.1.0",
```

Comment:
is this really needed? From my tests, just adding `http-proxy-agent` solves the issues

Reply:
I have removed `agent-base` and looks like it is not needed. Thanks for the reivew